polonius-engine/src/output/location_insensitive.rs

```diff
@@ -64,22 +66,28 @@ pub(super) fn compute<Region: Atom, Loan: Atom, Point: Atom>(
             all_facts.borrow_region.iter().map(|&(r, b, _p)| (r, b)),
         ));
 
-        region_live_at.insert(all_facts.region_live_at.into());
-
-        invalidates.insert(Relation::from(
-            all_facts.invalidates.iter().map(|&(p, b)| ((b, p), ())),
-        ));
-
         // .. and then start iterating rules!
         while iteration.changed() {
             // requires(R2, B) :- requires(R1, B), subset(R1, R2).
+            //
+            // Note: Since `subset` is effectively a static input, this join can be ported to
+            // a leapjoin. Doing so, however, was 7% slower on `clap`.
```

Comment:
Makes sense: I imagine that a plain "join" is more efficient if there are exactly two inputs.